chaosaws/elasticache/probes.py

```diff
@@ -0,0 +1,115 @@
+# -*- coding: utf-8 -*-
+from botocore.exceptions import ClientError
+from chaoslib.exceptions import FailedActivity
+from chaoslib.types import Configuration, Secrets
+
+from chaosaws import aws_client
+from chaosaws.types import AWSResponse
+
+__all__ = ['describe_cache_cluster', 'get_cache_node_count',
+           'get_cache_node_status']
+
+
+def describe_cache_cluster(cluster_id: str,
+                           show_node_info: bool = False,
+                           configuration: Configuration = None,
+                           secrets: Secrets = None) -> AWSResponse:
+    """Returns cache cluster data for given cluster
+
+    :param cluster_id: str: the name of the cache cluster
```

Comment:
I think we will eventually need to clarify the right docstring format so that they look right on the docs website.